frontend/src/metabase-types/api/dataset.ts

```diff
@@ -194,7 +194,7 @@ export interface TemplateTag {
   dimension?: LocalFieldReference;
   "widget-type"?: string;
   required?: boolean;
-  default?: string | null;
+  default?: string | string[] | null;
```

Comment:
I see the actual API call, it could be `string[]` as well. I think we might not use the type before, so it's outdated.

Reply:
Unfortunately, this failed type in another location. And I don't wanna mess around with query builder code.